docs/examples/mxnet/mxnet-resnet50.ipynb

```diff
@@ -200,22 +200,22 @@
-      "INFO:root:Epoch[0] Batch [100]\tSpeed: 5381.66 samples/sec\taccuracy=0.001257\n",
-      "INFO:root:Epoch[0] Batch [200]\tSpeed: 6635.37 samples/sec\taccuracy=0.001445\n",
-      "INFO:root:Epoch[0] Batch [300]\tSpeed: 6744.02 samples/sec\taccuracy=0.003154\n",
-      "INFO:root:Epoch[0] Batch [400]\tSpeed: 6767.70 samples/sec\taccuracy=0.006914\n",
-      "INFO:root:Epoch[0] Batch [500]\tSpeed: 6787.36 samples/sec\taccuracy=0.010078\n",
-      "INFO:root:Epoch[0] Batch [600]\tSpeed: 6796.49 samples/sec\taccuracy=0.016934\n",
-      "INFO:root:Epoch[0] Batch [700]\tSpeed: 6816.37 samples/sec\taccuracy=0.023477\n",
-      "INFO:root:Epoch[0] Batch [800]\tSpeed: 6801.38 samples/sec\taccuracy=0.032256\n",
-      "INFO:root:Epoch[0] Batch [900]\tSpeed: 6826.82 samples/sec\taccuracy=0.042510\n",
-      "INFO:root:Epoch[0] Batch [1000]\tSpeed: 6812.82 samples/sec\taccuracy=0.052744\n",
-      "INFO:root:Epoch[0] Batch [1100]\tSpeed: 6817.72 samples/sec\taccuracy=0.063359\n",
-      "INFO:root:Epoch[0] Batch [1200]\tSpeed: 6828.04 samples/sec\taccuracy=0.069570\n",
-      "INFO:root:Epoch[0] Train-accuracy=0.081150\n",
-      "INFO:root:Epoch[0] Time cost=202.865\n",
+      "INFO:root:Epoch[0] Batch [100]\tSpeed: 4407.30 samples/sec\taccuracy=0.001141\n",
+      "INFO:root:Epoch[0] Batch [200]\tSpeed: 4444.77 samples/sec\taccuracy=0.003184\n",
+      "INFO:root:Epoch[0] Batch [300]\tSpeed: 4395.88 samples/sec\taccuracy=0.006074\n",
+      "INFO:root:Epoch[0] Batch [400]\tSpeed: 4384.70 samples/sec\taccuracy=0.011182\n",
+      "INFO:root:Epoch[0] Batch [500]\tSpeed: 4389.42 samples/sec\taccuracy=0.017441\n",
+      "INFO:root:Epoch[0] Batch [600]\tSpeed: 4382.10 samples/sec\taccuracy=0.026377\n",
+      "INFO:root:Epoch[0] Batch [700]\tSpeed: 4388.26 samples/sec\taccuracy=0.036611\n",
+      "INFO:root:Epoch[0] Batch [800]\tSpeed: 4383.51 samples/sec\taccuracy=0.047139\n",
+      "INFO:root:Epoch[0] Batch [900]\tSpeed: 4402.73 samples/sec\taccuracy=0.057686\n",
+      "INFO:root:Epoch[0] Batch [1000]\tSpeed: 4392.32 samples/sec\taccuracy=0.067861\n",
+      "INFO:root:Epoch[0] Batch [1100]\tSpeed: 4384.42 samples/sec\taccuracy=0.079248\n",
+      "INFO:root:Epoch[0] Batch [1200]\tSpeed: 4385.37 samples/sec\taccuracy=0.090088\n",
```

Comment:
What is this speed performance drop? Different # of GPUs?

Reply:
Previous number may be generated somewhere else, but it would be good to double check if without that changes @klecki  gets the same numbers.

Reply:
This is related to the machine I was using, I checked the before and after and it's consistent (even slower, cause almost everything was occupied, so I took something slower).